FAQ.md

```diff
@@ -0,0 +1,15 @@
+# FAQs
+
+## Why balenaSound?
+We wanted to create an open source alternative to existing audio streaming commercial products that was not tied to a vendor or platform. balenaSound is the product of that. balenaSound is free, secure and fully customizable! You can pick the features that suit you and leave those that don't behind, resting asured that your device will keep on working no matter what.
+
+## Is balenaSound free?
+Yes! balenaSound requires you to sign up for a free [balenaCloud](https://dashboard.balena-cloud.com/signup) account. Your first 10 devices are always
+free and full-featured.
+
+## Privacy: is balenaSound secure? 
+balenaSound does not not depend on any type of cloud service to deliver a great experience. Other than that, your device will be running balenaOS which does communicate with balenaCloud to get application updates. This can be disabled however and won't affect your experience. Both [balenaSound](https://github.com/balenalabs/balena-sound/) and [balenaOS](https://github.com/balena-os) are open source projects, so feel free to check the source code if you want to know more about how they work!
+
+## Does balenaSound need internet access?
```

Comment:
Isn't on the deployed site. Don't ask me why.

Reply:
I mean internet access.

Reply:
For some reason landrbot is not picking up the updates. I know they were changing the server where it's hosted, so might be that. Already notified about it, guess we need to wait :)